packages/yourloops/test/integration/assert/stats.assert.ts

```diff
@@ -72,7 +72,7 @@ export const checkReadingsInRangeStats = async () => {
 }
 
 export const checkReadingsInRangeStatsWidgets = async () => {
-  const patientStatistics = within(await screen.findByTestId('patient-statistics', {}, { timeout: 3000 }))
+  const patientStatistics = within(await screen.findByTestId('patient-statistics', {}, { timeout: 4000 }))
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Is this change necessary or did you forget to remove it?